src/Asset/Capacity/CapacityInterface.php

```diff
@@ -62,6 +62,22 @@ public function getSearchOptions(string $classname): array;
      */
     public function getSpecificRights(): array;
 
+    /**
+     * Check if the capacity can be safetly disabled.
+     *
+     * @param \CommonDBTM $asset
+     * @return bool
+     */
+    public function canBeDisabled(\CommonDBTM $asset): bool;
```

Comment:
It will anyway be possible to disable it. `isUsed` will be less ambiguous about that.

```suggestion
    public function isUsed(\CommonDBTM $asset): bool;
```

Also, as the capacity is not related to a specific asset instance, but to all the assets from a definition, the method parameter should be `string $classname`.